net/upnp/src/www/services_upnp.php

```diff
@@ -255,6 +268,44 @@ function miniupnpd_validate_port($port)
                        </div>
                       </td>
                     </tr>
+
+                    <tr>
+                        <td><a id="help_for_override_subnet" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Interface subnet override");?></td>
+                        <td>
+                            <select name="overridesubnet" class="selectpicker" id="overridesubnet">
+                                <option value="" <?=!empty($pconfig['overridesubnet']) ? "selected=\"selected\"" : "";?>>Interface default</option>
+<?php
+                                for ($i = 32; $i >= 1; $i--) { ?>
+                                    <option value="/<?=$i;?>" <?=!empty($pconfig['overridesubnet']) && $pconfig['overridesubnet'] == "/".$i ? "selected=\"selected\"" : "";?>>/<?=$i;?></option>
```

Comment:
it's probably safer to validate for subnet mask and add the "/" in the configuration. /32 is basically the default like when we omit the override?